.travis.yml

```diff
@@ -1,9 +1,12 @@
-language: generic
+language: node_js
+node_js:
```

Comment:
It's a little weird to make this a node project, but that was the only way I found to control both the node version and the emacs version.